ouroboros-consensus/src/Ouroboros/Consensus/Block/Abstract.hs

```diff
@@ -26,6 +27,29 @@ module Ouroboros.Consensus.Block.Abstract (
   , decodeRawHash
     -- * Existentials
   , SomeBlock(..)
+    -- * Re-export basic definitions from @ouroboros-network@
+  , blockMeasure
+  , BlockMeasure -- opaque
+  , blockPoint
+  , castHash
+  , castPoint
+  , ChainHash(..)
+  , HasHeader(..)
+  , HeaderHash
+  , Point(GenesisPoint, BlockPoint)
+  , pointHash
+  , pointSlot
+  , StandardHash
+    -- * Re-export basic definitions from @cardano-base@
```

Comment:
Actually, `cardano-slotting`, which lives in the `slotting` dir of `cardano-base` :grin:  (who needs package and folder names to match? :roll_eyes:)

Reply:
Let's say it's the repo name :)